ivadomed/loader/utils.py

```diff
@@ -296,6 +297,9 @@ def get_new_subject_split_new(df, split_method, data_testing, random_seed,
     split_path = os.path.join(path_output, "split_datasets.joblib")
     joblib.dump(split_dct, split_path)
 
+    # Save the newly created .tsv on the output folder to be used during evaluation
```

Comment:
Here, you save the df in the `get_new_subject_split_new` function. This function is part of the new loader project and not integrated in the pipeline yet. You may want to move the save in the `get_new_subject_split` function above where you merge the 2 datasets.